test/integration/com/unifina/com/unifina/domain/marketplace/PaidSubscriptionIntegrationSpec.groovy

```diff
@@ -30,6 +30,8 @@ class PaidSubscriptionIntegrationSpec extends IntegrationSpec {
 			ownerAddress: "0x0000000000000000000000000000000000000000",
 			beneficiaryAddress: "0x0000000000000000000000000000000000000000",
 			pricePerSecond: 1,
+			contact: new Product.Contact(),
```

Comment:
This is all over the Specs now, can this kind of changes be avoided by initializing the fields with default values (see previous comment)?